aws-rds-dbcluster/src/main/java/software/amazon/rds/dbcluster/CallbackContext.java

```diff
@@ -3,19 +3,38 @@
 import software.amazon.cloudformation.proxy.StdCallbackContext;
 import software.amazon.rds.common.handler.TaggingContext;
 
+import java.util.HashMap;
+import java.util.Map;
+
 @lombok.Getter
 @lombok.Setter
 @lombok.ToString
 @lombok.EqualsAndHashCode(callSuper = true)
 public class CallbackContext extends StdCallbackContext implements TaggingContext.Provider {
     private boolean modified;
+    private boolean isRebooted;
```

Comment:
NIT: please avoid modal verbs in var names here: a Boolean variable would trigger lombook to generate the corresponding `isRebooted` method.